server/index.js

```diff
@@ -14,6 +14,8 @@ const port = 3000;
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 app.use(express.static(path.join(__dirname, '/../public')));
+app.use(express.json());
```

Comment:
This looks great! Can you uninstall bodyParser and remove line 14 & 15? I think your way is better since it's built in & we don't need duplicate  code.

Reply:
oh, sorry, just realized they're doing the same thing. And just uninstalled it